basics/dataproc_cluster/stable/main.tf

```diff
@@ -0,0 +1,59 @@
+#resource "google_service_account" "default" {
+#  account_id   = "service-account-id"
+#  display_name = "Service Account"
+#}
+
+resource "google_dataproc_cluster" "mycluster" {
+  name     = "dataproc-cluster"
+  region   = "us-central1"
+  graceful_decommission_timeout = "120s"
+
+  cluster_config {
+    staging_bucket = var.gcp_project_id
```

Comment:
Bucket names should not be set as project id! Doing so means the student can only have one bucket.
The best practice convention is to use ${PROJECT_ID}-${NAME}